### DISCUSSION TITLE
Extract Metadata since modified option

### DISCUSSION THREAD

**Discussion by dsm1212** [immich-app/immich#5763] [Feature Request]
### The feature

Sometimes I re-export already processed photos from lightroom with more metadata. The extract meta data (and really all the jobs) need a "modified" button. "all" and "missing" don't work for updated files. As my library gets bigger "all" gets more and more painful.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
I am assuming you are talking about external library?

**Comment by dsm1212** [comment]
Sorry, yes right
